lib/datadog/appsec/configuration/settings.rb

```diff
@@ -128,6 +128,10 @@ def ruleset
           @options[:ruleset]
         end
 
+        def ip_denylist
```

Comment:
This configurable is not meant to be publicly used, but is very useful for testing.

I wish we had a way to mark such things as experimental to kind-of feature-flag these behind a very obvious "you can try this but you're on your own and it may change at any point in time".

Reply:
> I wish we had a way to mark such things as experimental to kind-of feature-flag these behind a very obvious "you can try this but you're on your own and it may change at any point in time".

Add a comment to this method with such explanation :)

Reply:
OK, done!